src/diagnostics/land_compute_methods.jl

```diff
@@ -139,7 +139,20 @@ end
 @diagnostic_compute "vapor_flux" SoilCanopyModel p.soil.turbulent_fluxes.vapor_flux
 
 # Soil - SoilCO2
-@diagnostic_compute "heterotrophic_respiration" SoilCanopyModel p.soilco2.top_bc
+function compute_heterotrophic_respiration!(
+    out,
+    Y,
+    p,
+    t,
+    land_model::SoilCanopyModel{FT},
+) where {FT}
+    if isnothing(out)
+        return p.soilco2.top_bc .* FT(83.26)
```

Comment:
Maybe we could note that:
# To convert from kg C to mol CO2, we need to multiply by: [3.664 kg CO2/ kg C] x [10^3 g CO2/ kg CO2] x [1 mol CO2/44.009 g CO2] = 83.26 mol CO2/kg C

Reply:
Technically 83.26 is a parameter in ClimaParams. Would it be possible to fetch its value from `land_model`?

Reply:
I wonder if I should remove it from ClimaParams, as it is just a conversion?

Reply:
that I added but actually we are not using it (except if we do use it here)

Reply:
To make it makes sense that it should not be a parameter (but there are other constants of nature/conversions in there). I'd remove it.